### PR TITLE
Update c3 amended

### PR DESCRIPTION
**Pull request by kodster28**
replaces #12966

### CODE REVIEW

File: content/workers/get-started/guide.md [hunk 1]
```diff
@@ -62,6 +62,31 @@ In your project directory, C3 has generated the following:
 4. `package-lock.json`: Refer to [`npm` documentation on `package-lock.json`](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json).
 5. `node_modules`: Refer to [`npm` documentation `node_modules`](https://docs.npmjs.com/cli/v7/configuring-npm/folders#node-modules).
 
+### Create a new Worker Project from an external source
+
+In addition to creating new projects from C3 templates, C3 also supports creating new projects from Git repositories. To create a new project from a Git repository, open your terminal and run:
+
+```sh
+$ npm create-cloudflare@latest -- --template [<SOURCE>]
```
Comment (pedrosousa) [inline content/workers/get-started/guide.md]: Is `<SOURCE>` optional? If it's mandatory, I believe we should remove the `[]`, both here and in line 73.

File: content/workers/get-started/guide.md [hunk 2]
```diff
@@ -62,6 +62,31 @@ In your project directory, C3 has generated the following:
...
+
+`[<SOURCE>]` may be any of the following:
+
+- user/repo
+- git@github.com:user/repo
+- https://github.com/user/repo
+- user/repo/some-template (subdirectories)
+- user/repo#canary (branches)
+- user/repo#1234abcd (commit hash)
+- bitbucket:user/repo (BitBucket)
```
Comment (pedrosousa) [inline content/workers/get-started/guide.md]:
```suggestion
- bitbucket:user/repo (Bitbucket)
```

File: content/workers/get-started/guide.md [hunk 3]
```diff
@@ -62,6 +62,31 @@ In your project directory, C3 has generated the following:
...
+
+In addition to creating new projects from C3 templates, C3 also supports creating new projects from Git repositories. To create a new project from a Git repository, open your terminal and run:
+
+```sh
+$ npm create-cloudflare@latest -- --template [<SOURCE>]
+```
+
+`[<SOURCE>]` may be any of the following:
+
+- user/repo
```
Comment (pedrosousa) [inline content/workers/get-started/guide.md]:
```suggestion
- user/repo (GitHub)
```

File: content/workers/get-started/quickstarts.md [hunk 4]
```diff
@@ -15,23 +15,23 @@ meta:
 Quickstarts are GitHub repositories that are designed to be a starting point for building a new Cloudflare Workers project. To start any of the projects below, run:
 
 ```sh
-$ npx wrangler generate <NEW_PROJECT_NAME> <GITHUB_REPO_URL>
+$ npm create cloudflare@latest <NEW_PROJECT_NAME> -- --template [<GITHUB_REPO_URL>]
```
Comment (pedrosousa) [inline content/workers/get-started/quickstarts.md]: Same comment about the `[]` characters in `[<GITHUB_REPO_URL>]`.

**Review by pedrosousa — APPROVED** [review]

File: content/workers/get-started/guide.md (hunk 1 above)
Comment (kodster28) [inline content/workers/get-started/guide.md]:
```suggestion
$ npm create cloudflare@latest -- --template [<SOURCE>]
```

**Pull request merged**